README.md

```diff
@@ -41,7 +41,7 @@ NB: If you don't have any Meilisearch instance running and containing your data,
 - [🔧 Installation](#-installation)
 - [🎬 Usage](#-usage)
 - [💅 Customization](#-customization)
-- [⚡️ Example with InstantSearch](#-example-with-instantSearch)
+- [⚡️ Example with InstantSearch](#-example-with-instantsearch)
```

Comment:
This one still does not work :/  These seems to be a weird behavior when I look at the anchor. 

> %EF%B8%8F-example-with-instantsearch

Could you fix it?

Reply:
I will look into that, sure.
The behavior is weird, so probably will look into how github handles the anchor links and emoji.

Reply:
Feel free to change the emoji if it is too annoying. You can chose 🔥